cmd/oras/pull.go

```diff
@@ -103,17 +104,38 @@ func runPull(opts pullOptions) error {
 
 	// Copy Options
 	copyOptions := oras.DefaultCopyOptions
+	configFileName, configMediaType := parseFileRef(opts.ManifestConfigRef, oras.MediaTypeUnknownConfig)
+	copyOptions.FindSuccessors = func(ctx context.Context, fetcher content.Fetcher, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		successors, err := content.Successors(ctx, fetcher, desc)
+		if err != nil {
+			return nil, err
+		}
+		ret := []ocispec.Descriptor{}
```

Comment:
See https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices
```suggestion
		var ret []ocispec.Descriptor
```

Reply:
I was wondering if nil slice works for append, tested and looks good.